### ISSUE TITLE
Mercadopago (LATAM payment gateway)

### ISSUE THREAD

**Issue by joaquinv98** [Attendize/Attendize#1035]
I was trying to make [this](https://github.com/lucassmacedo/omnipay-mercadopago) work in Attendize, but I'm having some issues to do it.

Following the "guide" or steps posted in #521 

But I still don't get what or where should I modify the code exactly, if someone can give me a little help I would appreciate it

**Comment by dmb1987** [comment]
could you? , I need to implement it too and I can't

**Comment by joaquinv98** [comment]
Yes, I did it like 6 months ago, I do not remember exactly how, and it has some errors in the callback (some transactions might not be registered or marked as paid)

I could share the whole website but first I will need to clean it from all customer information, that will take me time because I'm very bussy rn.

**Comment by dmb1987** [comment]
great, I understand you're busy, it's not something urgent, I think that instead of the entire site if you could share how the payment_gateways table and the app/Http/Controllers/EventCheckoutController.php (and config/attendize.php) files look like great help. Thanks for answering!
